cli/command/registry_test.go

```diff
@@ -61,7 +61,7 @@ func TestGetDefaultAuthConfig(t *testing.T) {
 	}
 	for _, tc := range testCases {
 		serverAddress := tc.inputServerAddress
-		authconfig, err := command.GetDefaultAuthConfig(cfg, tc.checkCredStore, serverAddress, serverAddress == "https://index.docker.io/v1/")
+		authconfig, err := command.GetDefaultAuthConfig(cfg, tc.checkCredStore, serverAddress)
```

Comment:
It's a bit unfortunate that some of these are exported; we may need to do a proper assessment to learn how widely these may be used (and thus; how breaking changing the signature is).

Reply:
I mean, we can revert it. It's an entirely unnecessary/clunky parameter – the information is redundant and passing flags like this is always a bit of a smell

Reply:
Yes, I had in mind that if this is to be included in a minor release, we need to be careful not to introduce a breaking change. Agreed that most of this is iffy / redundant, just trying to reduce risk

That said, the signature of `ConfigureAuth` was also changed, which may be more difficult to keep the same as before (unless we do a new function for that I guess)

Reply:
Reverted the signature changes here (and in `ConfigureAuth`).